lib/core/client.js

```diff
@@ -487,7 +487,7 @@ class NightwatchClient extends EventEmitter {
       this.settings.webdriver.default_path_prefix = this.transport.defaultPathPrefix;
     }
 
-    if (this.argv.parallel && this.settings.webdriver.start_process) {
+    if (this.settings.testWorkersEnabled && this.settings.webdriver.start_process) {
```

Comment:
can we please add tests as well

Reply:
port assignment has to be random for multiple webdriver instances to run simultaneously.

Reply:
we still need a small test here.